test-it/ExpectedResults/Log10.vtr

```diff
@@ -0,0 +1,92 @@
+------ DBL --------
+Log10(0) => -Inf
+Log10(1) => 0
+Log10(10) => 1
+Log10(1012) => 3.00518
+Log10(3.1412) => 0.497096
+Log10(-1) => NaN
+Log10(Inf) => Inf
+Log10(-Inf) => NaN
+Log10(NaN) => NaN
+------ SGL --------
+Log10(0) => -Inf
+Log10(1) => 0
+Log10(10) => 1
+Log10(1012) => 3.00518
+Log10(3.1412) => 0.497096
+Log10(-1) => NaN
+Log10(Inf) => Inf
+Log10(-Inf) => NaN
+Log10(NaN) => NaN
+------ INT --------
+Log10(0) => -Inf
+Log10(1) => 0
+Log10(10) => 1
+Log10(1012) => 3.00518
+Log10(3) => 0.477121
+Log10(-1) => NaN
+Log10(2.14748E+09) => 9.33193
+------ CDB --------
```

Comment:
It is a good idea to do epsilon comparison for floating point than using expected values (to make sure they continue to pass in different platforms, compilers, so forth).

Reply:
I agree. In this case, Printf seems to output the right number of digits under both native and JS and hence it works.  I will address it when I submit the NXG acceptance tests.